.jenkins/pytorch/test.sh

```diff
@@ -144,6 +144,11 @@ test_aten() {
   fi
 }
 
+test_numpy_uninstall() {
+  conda uninstall numpy
```

Comment:
This should give a `Proceed ([y]/n)?` prompt, so I think it needs `-y`.

Reply:
Also, this can be slow (~5 seconds even on a fast connection). Probably fine here, but if it's worth avoiding then you could create a dummy package named `numpy` that raises an `ImportError` and put it on the path.

Reply:
Thank you for the suggestion! It sounds much nicer!